tracer/src/Datadog.Trace.Tools.Runner/CheckAgentCommand.cs

```diff
@@ -24,7 +24,7 @@ public override async Task<int> ExecuteAsync(CommandContext context, CheckAgentS
             if (settings.Url == null)
             {
                 // Try to autodetect the agent settings
-                configuration = new ExporterSettings(new EnvironmentConfigurationSource(), NullConfigurationTelemetry.Instance);
+                configuration = new ExporterSettings(new EnvironmentConfigurationSource(true), NullConfigurationTelemetry.Instance);
```

Comment:
if we keep this, adding the parameter name could ease reading why we need to pass true